src/custom/state/swap/hooks.tsx

```diff
@@ -221,8 +223,8 @@ export function useDerivedSwapInfo(): DerivedSwapInfo {
     recipient,
   } = useSwapState()
 
-  const inputCurrency = useTokenBySymbolOrAddress(inputCurrencyId)
-  const outputCurrency = useTokenBySymbolOrAddress(outputCurrencyId)
+  const inputCurrency = useTokenBySymbolOrAddress(isSupportedNetwork ? inputCurrencyId : null)
+  const outputCurrency = useTokenBySymbolOrAddress(isSupportedNetwork ? outputCurrencyId : null)
```

Comment:
why not do it already in `useTokenBySymbolOrAddress`?

Reply:
Thanks! Fixed